spec/helpers/task_list_helper_spec.rb

```diff
@@ -2,6 +2,8 @@
 
 RSpec.describe TaskListHelper, type: :helper do
   describe "#task_status_id" do
+    before { mock_successful_api_establishment_response(urn: 12345) }
```

Comment:
The scope of this is a little wide, I think you felt like you shouldn't need this mocking and I don't think you do.

This helper simply takes a Task and renders a string from the title, to exercise that behaviour all we need is:

```ruby
      task = Task.new(title: "Clear land questionnaire")
      expect(helper.task_status_id(task)).to eq "clear-land-questionnaire-status"
```
```